docs/installation/ubuntulinux.md

```diff
@@ -267,7 +267,8 @@ enabled, you must set UFW's forwarding policy appropriately.
 
 Also, UFW's default set of rules denies all incoming traffic. If you want to be able
 to reach your containers from another host then you should also allow incoming
-connections on the Docker port (default `2375`).
+connections on the Docker port (defaults to `2376` is TLS is enabled, or `2375` for
```

Comment:
@SvenDowideit this is a bit over packed...unpacking will make it easier for folks. TLS enabled is the default right? Still not clear so we should say so.

--->

Also, UFW's default set of rules denies all incoming traffic. If you want to to
reach your containers from another host allow incoming connections on the Docker
port. The Docker port defaults to `2376` if TLS is enabled or `2375` when it is
not. If TLS is not enabled, communication is unencrypted.  By default, Docker runs without TLS enabled.